front/context/src/AccountsContext.tsx

```diff
@@ -14,6 +14,7 @@ import { Option } from '@polkadot/types';
 import { AccountId, StakingLedger } from '@polkadot/types/interfaces';
 import { logger } from '@polkadot/util';
 import { decodeAddress, encodeAddress } from '@polkadot/util-crypto';
+import { writeStorage } from '@substrate/local-storage';
```

Comment:
Oh, wasn't aware of this. Where is the source code?

Reply:
https://github.com/yjkimjunior/local-storage

I just forked the rehooks one and made some small edits to a lingering PR to enable it for SSR

Reply:
Just a thought, not tested: if it's only to make `gatsby build` pass, maybe something like https://www.npmjs.com/package/localstorage-polyfill could be enough